tools/dockerise/__init__.py

```diff
@@ -175,17 +206,16 @@ def run(rebuild, clean, platform, network, **kwargs):
 
                 # If we are requesting a rebuild, then run build
                 if rebuild:
-                    build_platform(platform)
+                    build_platform(platform, local_only)
                     # If we were building the selected platform we have to move our selected tag up
                     if selected_platform:
                         if (
                             subprocess.call(
-                                ["docker", "image", "tag", tag, "{}:selected".format(repository)],
-                                stdout=subprocess.DEVNULL,
+                                ["docker", "tag", tag, "{}:selected".format(repository)], stdout=subprocess.DEVNULL,
```

Comment:
```suggestion
                                ["docker", "tag", tag, f"{repository}:selected"], stdout=subprocess.DEVNULL,
```

Reply:
I feel like this is a change that should be made in a separate PR in all python files to maintain consistency

Reply:
My opinion is that it's only more readable in some situations though